pkg/agent/baker_test.go

```diff
@@ -1753,6 +1753,6 @@ var _ = Describe("getAKSGPUImageSHA", func() {
 		Expect(getAKSGPUImageSHA("standard_nv6ads_a10_v5")).To(Equal("sha-20ffa2"))
 	})
 	It("should use newest AKSGPUCudaSHA with non grid SKU", func() {
-		Expect(getAKSGPUImageSHA("standard_nc6_v3")).To(Equal("sha-e8873b"))
+		Expect(getAKSGPUImageSHA("standard_nc6_v3")).To(Equal("sha-16fd35"))
```

Comment:
oh, can we use the consts here as well?

Reply:
Are we sure we want it tho, see my last pr

Reply:
hmm I think it makes sense to use the consts. Similar to the [getGPUDriverVersion tests](https://github.com/Azure/AgentBaker/blob/master/pkg/agent/baker_test.go#L1735) above though, is there a reason we don't use the driver version const variables for each version?